even more scopes and launch edits (#388)
HL7/smart-app-launch: input/pages/scopes-and-launch-context.md

`openid fhirUser` | Permission to retrieve information about the current logged-in user.
`launch`          | Permission to obtain launch context when app is launched from an EHR.
`launch/patient`  | When launching outside the EHR, ask for a patient to be selected at launch time.

> `offline_access`  | Request a `refresh_token` that can be used to obtain a new access token to replace an expired one even after the end-user no longer is online after the access token expires.

Reviewer: ```suggestion
`offline_access`  | Request a `refresh_token` that can be used to obtain a new access token to replace an expired one, even after the end-user no longer is online after the access token expires.
```

For example:

> * If a client uses SMART App Launch to request `user/*.cruds` and is granted these scopes by a user, these scopes convey "full access" relative to the server's underlying permissions.  If the underlying user has limited permissions, the client will face these same limitations.

Reviewer: ```suggestion
* If a client uses SMART App Launch to request `user/*.cruds` and is granted these scopes by a user, these scopes convey "full access" relative to the user's underlying permissions.  If the underlying user has limited permissions, the client will face these same limitations.
```

The *user*'s permissions are being delegated. This language achieve consensus through pretty careful dissection; if we want to add to it, I'd suggest bringing a proposal back to FHIR-I.

Author: OK I will create a tracker,  I am not sure who the user is or what his role is.

> * If a client uses SMART Backend Services to request `system/*.cruds`, these scopes convey "full access" relative to a pre-configured server policy.  If the pre-configured policy imposes limited permissions, the client will face these same limitations.

Reviewer: ```suggestion
* If a client uses SMART Backend Services to request `system/*.cruds`, these scopes convey "full access" relative to a server's pre-configured client-specific policy.  If the pre-configured policy imposes limited permissions, the client will face these same limitations.
```

Does this clarify? The server maintains the policy about what the client can do.
